specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2018-11-19/machineLearningServices.json

```diff
@@ -404,6 +404,47 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.MachineLearningServices/locations/{location}/usagesbyvmfamily": {
```

Comment:
```suggestion
    "/subscriptions/{subscriptionId}/providers/Microsoft.MachineLearningServices/locations/{location}/usagesByVMFamily": {
```

Reply:
In my backend code, I have declared this route with "usagesbyvmfamily" in lower case. Will using the camel casing here affect that as well ?

Reply:
Sorry, I'm not sure

Reply:
Yes I believe so.  Per ARM guidelines this should be camel-cased so if the API hasn't been deployed yet I suggest changing it (we can also wait to see what the ARM reviewer says).

Reply:
Fixed it.